src/utils/greedy_grid/register.py

```diff
@@ -112,12 +111,6 @@ def register(source_point_cloud: np.ndarray,
                       (0.5 * voxel_size)
                       ])
 
-    t = t.clamp(-10, 10)
```

Comment:
@mariarzv it should not work (either `max` or `min` as was in the original proposal) as we are not limiting the translation we are just cutting the **optimal** one.